documents/TestSuite/_options.mtlx

```diff
@@ -8,9 +8,6 @@
         <!-- List of comma separated file names acts as a filter to only test documents with these names -->
         <parameter name="overrideFiles" type="string" value="" />
 
-        <!-- List of comma separated file names which require color management  -->
-        <parameter name="cmsFiles" type="string" value="color_management.mtlx" />
```

Comment:
This was not used anymore so removing the option.

Reply:
Is that because we create a color management system even when we don't need it?

Reply:
Yes, my thinking is that a CMS should always be available in our test suite so any test can have  colorspace attributes specified as needed. We should default to color management enabled since in real use-cases it will be used all the time.